content/post/mv-reflection-001.md

```diff
@@ -0,0 +1,46 @@
+
+This did lead me to ask myself a silly question: "is all research, when you really come down to it, an act of self-ethnography?"
+
+I think most researchers would answer this with a resounding no, but this style of research where we are in constant communication, sharing reflections and reflecting on each-others reflections leads me to think that maybe this structure does lend itself to deeply personal, self-referential work that includes a great deal of self-ethnography.
+
+So with that out of the way, what is it that I have been grasping for for so many years?
+
+The current question I'm asking myself is
+
+> How might we create a sustainable technology company that is controlled by its workers, distributes its returns equitably to them, and can scale?
+
+This leads immediately to three sub-questions:
+1. What does sustainable mean?
+2. What does it mean to be controlled by the workers?
+3. What does equitable mean here?
+4. What does "can scale" mean?
+
+I wrote down a sentence and then need to define almost every word in it. Research!!
+
+And in the spirit of all research being auto-ethnography, I should bring this back to myself. I've spent almost of all of my career working for small companies that I co-founded. And by small I mean me and a co-founder and maybe sometimes a contractor or two. This is a very high level of autonomy and control: as I often say when you're a couple of co-founders you de-factor are a worker-owned co-operative. This level of agency has been extremely fulfilling to me in my career, but obviously two co-founders doesn't really scale. I'd like to find a way to scale that feeling of agency and ownership while tackling problems that require large teams to solve. I'd like everyone that I work with to have that feeling of agency, not just the founding team.
+
+My primary goal is to figure this out. Worker-owned co-operatives are an obvious answer but I don't want to be dogmatic. And to be honest, even after doing the [Platform Coops MOOC](https://platform.coop/blog/platform-co-ops-now-2nd-edition/) I still have so so much to to learn about coops.
+
+My secondary goal is to get better at ~~writing~~ *thinking via writing*. To that end the fellowship is a great *container* in which to force myself to write a little bit more. We've discussed in our meetings the idea of me doing reflections via vlogs or voice notes, but the real truth is that I'd like to just be able to write more.
+
+What am I going to do?
+
+1) Reading: I'm going to start with Founders at Work to get a sense of what the co-op stage of startups was like
+2) Interviews: I have the idea of compiling some interviews with people who have "sustainable" tech companies.
+3) Writing: this reflection is hopefully the start of a regular writing practice.
```

Comment:
This is one reason I'm also excited about us trying out correspondences. There are so many *kinds* of writing, and I'm starting to notice that it's important I wield them strategically. These reflections are kind of to each other and kind of diaries and kind of to a nebulous "public" since we're putting them online. That collection of constraints leads to a particular style of writing and thinking.

Playing with other constraints / formats and looking at the difference in the style of thinking I think will be productive. And correspondence is a classic form for research in particular. The famous Royal Society, a key player in the scientific revolution, started as an ["invisible college"](https://en.wikipedia.org/wiki/Invisible_College)/["Republic of Letters"](https://en.wikipedia.org/wiki/Republic_of_Letters) and the republic of letters was specifically a community of handwritten correspondences that basically evolve into the modern idea of papers, journals, and peer review.

Perhaps this fellowship is ultimately a republic of letters...